retrieve-image-tags/config.json

```diff
@@ -73,6 +73,44 @@
       "idealista/prom2teams"
     ],
     "versionSource": "github-latest-release:idealista/prom2teams"
+  },
+  "vsphere-csi": {
+    "images": [
+      "gcr.io/cloud-provider-vsphere/csi/release/driver",
+      "gcr.io/cloud-provider-vsphere/csi/release/syncer"
+    ],
+    "versionSource": "github-latest-release:kubernetes-sigs/vsphere-csi-driver"
+  },
+  "csi-attacher": {
+    "images": [
+      "registry.k8s.io/sig-storage/csi-attacher"
+    ],
+    "versionSource": "github-latest-release:kubernetes-csi/external-attacher"
+  },
+  "csi-resizer": {
+    "images": [
+      "registry.k8s.io/sig-storage/csi-resizer"
+    ],
+    "versionSource": "github-latest-release:kubernetes-csi/external-resizer"
+  },
+  "storage-livenessprobe": {
+    "images": [
+      "registry.k8s.io/sig-storage/livenessprobe"
+    ],
+    "versionSource": "github-latest-release:kubernetes-csi/livenessprobe"
+  },
+  "csi-provisioner": {
+    "images": [
+      "registry.k8s.io/sig-storage/csi-provisioner"
+    ],
+    "versionSource": "github-latest-release:kubernetes-csi/external-provisioner"
+  },
+  "cpi-release-manager": {
+    "images": [
+      "gcr.io/cloud-provider-vsphere/cpi/release/manager"
+    ],
+    "versionSource": "github-latest-release:kubernetes/cloud-provider-vsphere",
+    "versionFilter": "^v1\\.[0-9]+\\.[0-9]+$"
```

Comment:
If you use `github-latest-release`, there won't be any filter applied as it will always just take the latest GitHub release. We can add the functionality if needed (for example, when multiple major/minor/patch are marked as latest whenever they release), but it isn't in there now.

Reply:
Ok then I think the best is to remove `github-latest-release` since we want to synchronize multiple images for different k8s versions and I don't know if they actually tag each one of them as latest at some point.

Reply:
I suspect they just release a batch of new tags for multiple minors at the same time, similar to how we do our releases. Are you going to remove this in favor of matching tags from Docker Hub, or?

Reply:
I replaced it with 
```
    "versionSource": "github-releases:kubernetes/cloud-provider-vsphere",
    "versionFilter": "^v1\\.[2-9]+\\.[3-9]+$"
```
If I understand correctly, it should copy all the new images for the minors when they're published.